airflow/api_fastapi/openapi/v1-generated.yaml

```diff
@@ -1037,3 +1081,25 @@ components:
       - msg
       - type
       title: ValidationError
+    VariableResponse:
+      properties:
+        key:
```

Comment:
Why do we actually re turn the key in the response if it was in the request... sounds use-less... but anyway is also in previous API :-D

Reply:
Yes because it would mean adapting the response to each endpoint and request depending on the query params and path param to substract them from the response. That would be a lot of extra work, and the returned schema would dynamically change, that would be hard for the client to know what is the actual 'body' schema for that specific request.

So usually we don't bother with this even if the data is indeed duplicated.